app/components/trainees/confirmation/diversity/view.rb

```diff
@@ -0,0 +1,76 @@
+module Trainees
+  module Confirmation
+    module Diversity
+      class View < GovukComponent::Base
+        include SanitizeHelper
+
+        attr_accessor :trainee
+
+        def initialize(trainee:)
+          @trainee = trainee
+        end
+
+        def get_diversity_information_rows
```

Comment:
I don't think we need to prefix all the public methods with `get` eg `get_diversity_information_rows` could just be `diversity_information_rows`?